hyper/http20/connection.py

```diff
@@ -102,6 +104,9 @@ def __init__(self, host, port=None, secure=None, window_manager=None,
         else:
             self.host, self.port = host, port
 
+        # HTTP Version of the connection
+        self.version = HTTPVersion.http20
```

Comment:
Given that these are static and don't change, let's set these on the class object rather than do this at initialisation time.